config/q-expand.csv

```diff
@@ -232,22 +237,22 @@ QQ2F,"Security"
 QQ2G,"Partnerships"
 QQ3,"IT Portfolio"
 QQA,"Data & Analytics"
-QQB,"Public Experience"
```

Comment:
Do we need to have all of the nodes in a tree? It seems strange that we are keeping QQBA - QQBE but getting rid of their parent QQB?

Reply:
So the Excel file from GSA is kinda ugly and doesn't have every level. I'm guessing QQB is a level that doesn't exist in the sheet but does because TTS added it in the bot. It's probably just me missing this and didn't keep it.